docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,4169 @@
+### Cohere connector blueprint example for embedding:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Cohere Connector: embedding",
+  "description": "The connector to cohere embedding model",
+  "version": 1,
+  "protocol": "http",
+  "credential": {
+    "cohere_key": "<PLEASE ADD YOUR Cohere API KEY HERE>"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "https://api.cohere.ai/v1/embed",
+      "headers": {
+        "Authorization": "Bearer ${credential.cohere_key}"
+      },
+      "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"END\" }"
+    }
+  ]
+}
+```
+#### Sample response
+```json
+{
+  "connector_id": "XU5UiokBpXT9icfOM0vt"
+}
+```
+
+
+### Corresponding Predict request example:
+
+```json
+POST /_plugins/_ml/models/<ENTER MODEL ID HERE>/_predict
+{
+  "parameters": {
+    "prompt": ["Say this is a test"]
+  }
+}
+```
+
+#### Sample response
+```json
+{
+  "inference_results": [
+    {
+      "output": [
+        {
+          "name": "response",
+          "dataAsMap": {
+            "id": "39097276-d926-4ca1-92e6-d54a3c969d42",
+            "texts": [
+              "Say this is a test"
+            ],
+            "embeddings": [
+              [
+                -0.76953125,
+                -0.12731934,
+                -0.52246094,
+                -1.2714844,
+                1.8076172,
+                0.13842773,
```

Comment:
Let keep the first five embedding float values, then use `...` to repreent the others. It seems not helping too much by showing thousands of floats.